README.md

```diff
@@ -84,3 +86,21 @@ Author: Darren Haynes
 |:---:|:---:| --- |
 | __push()__ | `0(log)n` | We are not using standard list iteration with this binary heap. We check for the position of the parent of each child and swap them if need be. The distance between each child and its parent is halfed as we move closer to the head of the heap. This 'halving' gives us a `0(log)n` operation. |
 | __pop()__ | `0(log)n` | The initial pop is just `0(1)` and the subsequent reordering of the values has a worst case scenario of `0(log)n`, following the same pattern as the push method. |
+
+<br />
+
+## Data Structure 6: Simple Graph
+Author: Darren Haynes
+
+### Big O Notation
+| Method | Big 0 | Description |
+|:---:|:---:| --- |
+| __add_node()__ | `0(1)` | Adding a node to the graph uses the python dictionary `setter`. |
+| __add_edge()__ | `0(1)` | Adding a node to the graph uses the python dictionary `setter`. |
```

Comment:
`add_edge` is in fact `O(k)` in your current implementation, as you're checking for membership in a list of edges with the `in` operator.